src/porepy/models/solution_strategy.py

```diff
@@ -593,7 +599,11 @@ def compute_residual_norm(
             float: Residual norm.
 
         """
-        residual_norm = np.linalg.norm(residual) / np.sqrt(residual.size)
+        residual_norm = (
```

Comment:
In general, I think we have tried to avoid long single-expression conditional statements for readability. Is that correct, @keileg?

Reply:
No strong opinion here, and will follow your guideline after you settled this - this info in itself will be of interest to me. 

Indeed, writing out the if-else conditional, will not take more lines of code. I guess I am just somewhat a fan of these "one-liners".

Reply:
Yes, the preferrence is for explicit if-elses in such cases; for some of us, resolving these nested expressions can be mentally taxing.